python/pyspark/pandas/utils.py

```diff
@@ -368,7 +368,7 @@ def align_diff_frames(
         - full: `resolve_func` should resolve only common columns from 'this' and 'that' DataFrames.
             For instance, if 'this' has columns A, B, C and that has B, C, D, `this_columns` and
             'that_columns' in this function are B, C and B, C.
-        - left: `resolve_func` should resolve columns including that columns.
+        - left: `resolve_func` should resolve columns including that column.
```

Comment:
should resolve columns including that column.
or
should resolve columns including those columns.

Reply:
This is confusing. I think "that" is used as the name of one of the DataFrames. So it's trying to say "columns from 'that'". Without quotes it's weird. You can consistently quote 'that' here to clarify, or pick different names to begin with